test/common.jl

```diff
@@ -9,43 +9,49 @@ const DETATOL = 1e-3 * D * TRATIO
 # Random tolerance
 const RNDATOL = 5e-2 * D * TRATIO * 2
 
-# Hand-coded multivaite Gaussain
+# Hand-coded multivariate Gaussian
```

Comment:
Let's make a TODO note on improving this implementation here.
We either need to implement a more Distributions.jl like interface as it's currently done in https://github.com/xukai92/VecTargets.jl or directly use this package if we decide to register it.

Reply:
I guess it would make the most sense to just use https://github.com/xukai92/VecTargets.jl directly because otherwise it seems as though we would just end up implementing the same functionality that's in that package already. Have you tested https://github.com/xukai92/VecTargets.jl on CUDA yet?